docs/configuration.md

```diff
@@ -0,0 +1,238 @@
+# Configuration methods
+
+cibuildwheel can either be configured using environment variables, or from
+config file such as `pyproject.toml`.
+
+This page describes how to set options. For a full list of available options, see the [options reference](options.md).
+
+## Environment variables {: #environment-variables}
+
+Environment variables can be set in your CI config. For example, to configure
+cibuildwheel to run tests, add the following YAML to your CI config file:
+
+!!! tab "GitHub Actions"
+
+    > .github/workflows/*.yml ([docs](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables)) (can be global, in job, or in step)
+
+    ```yaml
+    env:
+      CIBW_TEST_REQUIRES: pytest
+      CIBW_TEST_COMMAND: "pytest {project}/tests"
```

Comment:
Should we update the for 3.0, now that `{project}/` isn't needed, but test-sources might be?

Reply:
Yeah, good idea on removing {project}. I think we don't need TEST_SOURCES though? It's more of a thing that can be discovered as/when it's needed?

Reply:
I've made the change to all instances in docs of "pytest {project}/tests". I also updated the tests, as we should probably be testing what we're documenting!

Reply:
If someone isn't using `src` structure, they may run into it. Probably should at least add a warning and maybe an upgrade guide. Not now though.